src/dbus-objects/team.c

```diff
@@ -545,6 +545,8 @@ __ni_objectmodel_team_link_watch_to_dict(const ni_team_link_watch_t *lw, ni_dbus
 			ni_dbus_dict_add_bool(dict, "send_always", lw->arp.send_always);
 		if (lw->arp.missed_max > 0)
 			ni_dbus_dict_add_uint32(dict, "missed_max", lw->arp.missed_max);
+		if (lw->arp.vlanid != UINT16_MAX)
+			ni_dbus_dict_add_uint16(dict, "vlanid", lw->arp.missed_max);
```

Comment:
```suggestion
			ni_dbus_dict_add_uint16(dict, "vlanid", lw->arp.vlanid);
```

Reply:
good catch. This brings me actually to a other question on `wicked show-xml`
Currently we would display it like:
```
<team>
    <debug_level>0</debug_level>
    <notify_peers>
      <count>1</count>
      <interval>4294967295</interval>
    </notify_peers>
    <mcast_rejoin>
      <count>1</count>
      <interval>4294967295</interval>
    </mcast_rejoin>
    <runner name="activebackup">
      <hwaddr_policy>same_all</hwaddr_policy>
    </runner>
    <link_watch_policy>any</link_watch_policy>
    <link_watch>
      ....
      </watch>
    </link_watch>
    <ports>
      ....
    </ports>
    <address>42:00:00:00:00:01</address>
  </team>
````
I would say, it is ok except the `<interval>4294967295</interval>`, which actually means it isn't set.
Going to change it.